src/app/shared/components/forms/custom-button/custom-button.component.scss

```diff
@@ -29,3 +35,35 @@ button {
   font-weight: 400;
   padding: 1.1rem 3rem;
 }
+
+.register-user {
```

Comment:
what is this. we already have small medium large. do not keep creating buttons for specific cases